ui/src/components/Views/Configs/MountPointForm.tsx

```diff
@@ -1,137 +1,187 @@
+import { ExecResult } from '@docker/extension-api-client-types/dist/v1';
 import {
-  Box,
   Button,
   CircularProgress,
   Dialog,
   DialogActions,
   DialogContent,
+  Divider,
   FormControl,
   MenuItem,
+  Paper,
   Select,
   Typography,
 } from '@mui/material';
 import React, { ReactElement, useEffect, useState } from 'react';
-import { LATEST_IMAGE } from '../../../constants';
-import { useDDClient, useMountPoint } from '../../../services';
-import { DockerImage } from '../../../types';
-import { DownloadProgress } from '../../Feedback';
+import {
+  getOSsFromBinary,
+  getUsersFromBinaryUnix,
+  getUsersFromBinaryWindows,
+  useDDClient,
+  useMountPoint,
+} from '../../../services';
+
+const ShrinkedCircularProgress = (): ReactElement =>  <CircularProgress size={20} sx={{ margin: 1}}/>;
```

Comment:
nit: indentation matters! ;)  
```suggestion
const ShrinkedCircularProgress = (): ReactElement => <CircularProgress size={20} sx={{ margin: 1 }}/>;
```